doc/lightning-getroute.7.md

```diff
@@ -47,6 +47,9 @@ is undefined.
 
 *maxhops* is the maximum number of channels to return; default is 20.
 
+*maxscannodes* is the maximum number of nodes to scan for finding route;
```

Comment:
I would express this as `*maxscannodes* sets an upper bound the number of nodes examined in route traversal, to limit time spend.  The default is no limit`.

Reply:
Good description! :-)